internal/pkg/envconfig/envconfig.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// TODO: there proboly already is a package for this... Where?
+
+package envconfig
+
+import (
+	"fmt"
+	"os"
+)
+
+func NoCache() bool {
+	env := os.Getenv("SINGULARITY_NO_CACHE")
+
+	fmt.Println("DEBIG: ENV: ", env)
```

Comment:
`DEBUG:` ?

Reply:
Opps, yeah, its supposed to be `debug` :smile: This is very much a WIP, I'll remove it soon!